pages/runtime/terminal.md

```diff
@@ -1,50 +1,37 @@
 # Using the Web Terminal
 
+To help you debug and develop your application on resin.io, the dashboard provides a hand webterminal. This gives you console access to your running [container][docker-container] on the device and allows you to test out small snippets of code or check some system logs on your device. To use this feature, navigate your application and select the device you want access to. In that devices menu page you will find the `>_ Terminal` menu item.
+
 __Note:__ Don't use this feature for production applications as it introduces security vulnerabilities. Use only for development versions of your applications and ensure you close the terminal session when you are done.
 
 ## Establishing a Terminal Session
 
-Resin.io uses [tty.js][tty.js] to give you direct access to your devices. A button will appear giving you access to this interface once your device is **online** and code is **pushed to it** (if the button is not visible beneath the logging window, ensure that both of these conditions are met):-
-
-![Terminal Session Button](/img/terminal-button.png)
-
-Once a terminal session is established you'll be presented with an area in which the terminal session will be shown and a button to open the terminal:-
+  In order for you to start a terminal session in your device [container][docker-container], you first need to ensure that your device is **online** and code is **pushed to it** and is running. If your container code crashes or ends quickly, it is not possible to attach a console to it. One option to keep your containers running is to enable the INITSYSTEM in your container. This can easily be done by creating a device environment variable called `INITSYSTEM` and setting its value to `on`.
```

Comment:
yeah, it can be either, I had just left it as device env var because it read better